### PR TITLE
Added info about movie version sorting

### PR DESCRIPTION
**Pull request by DesertCookie**
Updated wiki to reflect movie version sorting in 10.7 according to [this](https://github.com/jellyfin/jellyfin-web/pull/1237) change to jellyfin-web.

### CODE REVIEW

**Review by nielsvanvelzen — APPROVED** [review]

File: general/server/media/movies.md [hunk 1]
```diff
@@ -50,3 +49,14 @@ If labels are not added to the end of filenames, as shown above, each file will
 
 > [!Note]
 > To group media manually, long-click or right-click media to highlight then select additional media to merge. Use the new bar that appears to 'Group Versions'.
+
+### Order of Versions
+
+Movie versions are presented in an alphabetically sorted list. An exception applies to resolution names, which are sorted in descending order from largest to smalles resolution. A version name qualifies as a resolution name when ending with either a `p` or an `i`.
```
Comment (Maxr1998) [inline general/server/media/movies.md:55]:
```suggestion
Movie versions are presented in an alphabetically sorted list. An exception applies to resolution names, which are sorted in descending order from largest to smallest resolution. A version name qualifies as a resolution name when ending with either a `p` or an `i`.
```
Preferably, one would even write "highest to lowest resolution".

**Review by Maxr1998 — CHANGES_REQUESTED** [review]

File: general/server/media/movies.md (hunk 1 above)
Comment (DesertCookie) [reply general/server/media/movies.md]: Thanks for the suggestion. I agree with your change request.

**Review by DesertCookie — COMMENTED** [review]
Agreed and changed.

**Review by Maxr1998 — APPROVED** [review]

**Comment by Maxr1998** [comment]
Thank you! It seems that markdownlint still has some complains, I'd appreciate if you could address them:
```
general/server/media/movies.md:60 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "Examples of sorting"]
general/server/media/movies.md:61 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "* `1080p`, `2160p`, `360p`, `4..."]
```

**Comment by DesertCookie** [comment]
> `general/server/media/movies.md:60 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "Examples of sorting"]`

What is that supposed to mean? Should I not use bold in markdown?

**Comment by h1dden-da3m0n** [comment]
> > `general/server/media/movies.md:60 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "Examples of sorting"]`
> 
> What is that supposed to mean? Should I not use bold in markdown?

I think the "MarkdownLinter" wants you to use `#### ...` instewad of you using just `bold` as a heading

File: general/server/media/movies.md [hunk 2]
```diff
@@ -50,3 +49,14 @@ If labels are not added to the end of filenames, as shown above, each file will
...
 > To group media manually, long-click or right-click media to highlight then select additional media to merge. Use the new bar that appears to 'Group Versions'.
+
+### Order of Versions
+
+Movie versions are presented in an alphabetically sorted list. An exception applies to resolution names, which are sorted in descending order from highest to lowest resolution. A version name qualifies as a resolution name when ending with either a `p` or an `i`.
+
+> [!Note]
+> The first movie version in the list is the one selected by default.
+
+**Examples of sorting**
```
Comment (h1dden-da3m0n) [inline general/server/media/movies.md:60]:
```suggestion
#### Examples of sorting
```
If I read the "MarkdownLinter" right thenn this is what it expects you to use :thinking:

Comment (Maxr1998) [reply general/server/media/movies.md:60]: That's how I'd understand it as well.

Comment (DesertCookie) [reply general/server/media/movies.md]: Thanks.

**Review by Maxr1998 — APPROVED** [review]

**Review by h1dden-da3m0n — APPROVED** [review]

**Pull request merged**